src/script/components/message/CallTimeoutMessage.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import NamedIcon from 'Components/NamedIcon';
+import {t} from 'Util/LocalizerUtil';
+import {CallingTimeoutMessage} from '../../entity/message/CallingTimeoutMessage';
+
+import {registerReactComponent, useKoSubscribable} from 'Util/ComponentUtil';
+import MessageTime from './MessageTime';
+import {REASON} from '@wireapp/avs';
+
+export interface CallTimeoutMessageProps {
+  message: CallingTimeoutMessage;
+}
+
+const CallTimeoutMessage: React.FC<CallTimeoutMessageProps> = ({message}) => {
+  const reason = message.reason;
+  const timestamp = useKoSubscribable(message.timestamp);
+
+  return (
+    <div className="message-header">
+      <div className="message-header-icon message-header-icon--svg">
+        <div className="svg-red">
+          <NamedIcon name="hangup-icon" width="20" height="8" />
+        </div>
+      </div>
+      <div
+        className="message-header-label"
+        data-uie-name="element-message-call"
+        data-uie-value={reason === REASON.NOONE_JOINED ? 'no-one-joined' : 'everyone-left'}
+      >
+        <span className="ellipsis">
+          {t('callWasEndedBecause')}{' '}
```

Comment:
```suggestion
          {t('callWasEndedBecause')}{' '}
```

This looks a bit odd. I guess you just want to connect `callWasEndedBecause` sentence with another translation through a space. I suggest adding a new `const` which holds the sentence, so that you can use `{text}` in the `render` method.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10709/commits/a9008bd2f03817cdf54a518f81ca14767041c40f